### DISCUSSION TITLE
esp32 ota compiling error

### DISCUSSION THREAD

**Discussion by Camp78** [micropython/micropython#9621] [General]
Hi, I've an error during compiling for esp32 ota.
/home/camp/micropython/ports/esp32/build-GENERIC_OTA/frozen_content.c:71:5: error: redeclaration of enumerator 'MP_QSTR_lock'
     MP_QSTR_lock,
     ^~~~~~~~~~~~
In file included from /home/camp/micropython/py/obj.h:33,
                 from /home/camp/micropython/py/objint.h:30,
                 from /home/camp/micropython/ports/esp32/build-GENERIC_OTA/frozen_content.c:16:
/home/camp/micropython/ports/esp32/build-GENERIC_OTA/genhdr/qstrdefs.generated.h:759:6: note: previous definition of 'MP_QSTR_lock' was here
 QDEF(MP_QSTR_lock, 9134, 4, "lock")
      ^~~~~~~~~~~~

**Comment by robert-hh** [comment]
Try to run first:

make BOARD=GENERIC_OTA clean

and then build again.

**Reply by Camp78** [reply]
It worked, thank's a lot for fast reply and exact Solution.
